README.md

```diff
@@ -1,14 +1,69 @@
+
+
+(Note - include the icons for the stack + team members)
```

Comment:
Please remove this note since you've already included the icons and team members.

Reply:
Done